### PR TITLE
chore(deps): update dependencies

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

**Comment by open-dynaMIX** [comment]
@winged Not really sure why [this commit](https://github.com/projectcaluma/caluma/pull/1981/commits/c7f6d73436d5e97b177134e55046b180c101edb8) is needed.

File: caluma/caluma_form/tests/test_answer.py [hunk 1]
```diff
@@ -252,7 +253,7 @@ def test_save_default_answer_python_api(
         answer = api.save_default_answer(question, user=admin_user, value=inp["value"])
         snapshot.assert_match(answer)
     else:
-        with pytest.raises(CustomValidationError):
+        with pytest.raises((CustomValidationError, ValidationError)):
```
Comment (winged) [inline caluma/caluma_form/tests/test_answer.py:256]: You can ditch the `CustomValidationError` here, as it's a subclass of DRF's `ValidationError`.

File: caluma/caluma_form/tests/test_document.py [hunk 2]
```diff
@@ -865,7 +866,7 @@ def test_save_document_answer(  # noqa:C901
             )
             snapshot.assert_match(answer)
         else:
-            with pytest.raises(Exception):
+            with pytest.raises((CustomValidationError, ValidationError)):
```
Comment (winged) [inline caluma/caluma_form/tests/test_document.py:869]: Same here, this is a redundancy: `issubclass(CustomValidationError, ValidationError) == True`

**Review by winged — CHANGES_REQUESTED** [review]
LGTM! Just some minor issue with the exception handling code changes

File: caluma/caluma_form/tests/test_answer.py (hunk 1 above)
Comment (open-dynaMIX) [reply caluma/caluma_form/tests/test_answer.py]: done

File: caluma/caluma_form/tests/test_document.py (hunk 2 above)
Comment (open-dynaMIX) [reply caluma/caluma_form/tests/test_document.py]: done

**Review by winged — APPROVED** [review]

**Comment by anehx** [comment]
@open-dynaMIX thanks for finishing this! :heart:

**Pull request merged**